src/applications/disability-benefits/686c-674/tests/e2e/test-helpers.js

```diff
@@ -487,133 +487,6 @@ export const fillVeteranMarriageHistoryDetails = (client, data) => {
     );
 };
 
-export const fillDeceasedName = (client, data) => {
```

Comment:
Same here. This reverts work merged in here - https://github.com/department-of-veterans-affairs/vets-website/pull/12854/files